File: LeetCode/0441. Arranging Coins/Solution.cpp
/**
 * 1335 / 1335 test cases passed.
 * Runtime: 0 ms
 * Memory Usage: 5.9 MB 
 */
class Solution {
public:
    int arrangeCoins(int n) {
        return static_cast<int>((-1 + sqrt(1 + 8.0 * n)) / 2);
    }
};

File: LeetCode/0441. Arranging Coins/solution.py
"""
1335 / 1335 test cases passed.
Runtime: 28 ms
Memory Usage: 15 MB
"""
class Solution:
    def arrangeCoins(self, n: int) -> int:
        return int((-1 + math.sqrt(1 + 8.0 * n)) / 2)
